ui/js/dfv/src/fields/wysiwyg/tinymce.js

```diff
@@ -116,6 +116,7 @@ const TinyMCE = ( {
 					content_css: false,
 					setup: onSetup,
 					height: editorHeight,
+					wpautop: true,
```

Comment:
Since it's a setting, shouldn't this variable be defined elsewhere just like editorHeight?

Reply:
Normally yes, but wpautop is a display option for WYSIWYG fields, not an input option (at the moment)

Reply:
To elaborate, did you test unchecking the `wpautop` setting to see if this properly disabled this setting in the editor?

Reply:
> Normally yes, but wpautop is a display option for WYSIWYG fields, not an input option (at the moment)

Just checked, you can definitely disable the `wpautop` option from within the field config. If this shouldn't be possible we should remove the option from the WYSIWYG field.

Reply:
@JoryHogeveen , with or without wpautop, from my perspective, the data entered are "jumbled up" as shown on the screen capture. Not sure if that's what you mean.

Reply:
@papa-pup I think you are talking about something else here.
The field settings itself allow you to enable/disable wpautop. This should reflect in the React code as well as the front-end. Currently only the front-end is affected by this setting.

Reply:
The content gets jumbled up from the front forms without this patch because TinyMCE did not have the wpautop set to match the admin for some reason. Turning off wpautop in the Pods field setting only ever changed how output would get rendered but we can do more testing to see how it deals with proper HTML when you turn it off on both editors (admin/front), assuming it has p tags etc.

Reply:
Updated the PR to include wpautop directly with the TinyMCE editor (previously this wasn't used). That tests fine and works as expected.